tools/mitohifi/macros.xml

```diff
@@ -0,0 +1,22 @@
+<macros>
+    <token name="@TOOL_VERSION@">2</token>
+    <token name="@GALAXY_TOOL_VERSION@">galaxy0</token>
+    <xml name="edam_ontology">
+        <edam_topics>                                                                                  
+            <edam_topic>topic_0196</edam_topic>
```

Comment:
do you check thos topics?

Reply:
Yes, assembly https://bioportal.bioontology.org/ontologies/EDAM?p=classes&conceptid=topic_0196